tests/functional/cli/test_cli.py

```diff
@@ -64,7 +58,7 @@ def test_convert_single_file_in_place(nb_file, tmpdir):
     compare_notebooks(nb2, nb1)
 
 
-@requires_jupytext_installed
+@pytest.mark.requires_jupytext
```

Comment:
This mark is confusing

Reply:
Some of the tests require a local installation of `jupytext` to work, possibly the previous name was more explicit?

Reply:
In `src-layout` you can safely assume that the project is always run after a `pip install`, so I guess this can be removed.

Reply:
I have removed this one